lectures/heavy_tails.md

```diff
@@ -1036,26 +1081,41 @@ Use `np.random.seed(11)` to set the seed.
 
 ```{code-cell} ipython3
 from scipy.stats import pareto
+import numpy as np
+import matplotlib.pyplot as plt
 
 np.random.seed(11)
 
 n = 120
 alphas = [1.15, 1.50, 1.75]
+fig, axes = plt.subplots(3, 1, figsize=(8, 10))  # Increased size
 
-fig, axes = plt.subplots(3, 1, figsize=(6, 8))
-
-for (a, ax) in zip(alphas, axes):
+for ax, alpha in zip(axes, alphas):
     ax.set_ylim((-5, 50))
-    data = pareto.rvs(size=n, scale=1, b=a)
+    data = pareto.rvs(size=n, scale=1, b=alpha)
     ax.plot(list(range(n)), data, linestyle='', marker='o', alpha=0.5, ms=4)
     ax.vlines(list(range(n)), 0, data, lw=0.2)
-    ax.set_title(f"Pareto draws with $\\alpha = {a}$", fontsize=11)
+    ax.set_title(f'draws from Pareto ($\\alpha = {alpha}$)', fontsize=11)
```

Comment:
Please use the `α` and `αs` symbol in the code here and below.

In Jupyter lab, we can type `\alpha` and click on the tab bottom on the keyboard. It will give you `α` symbol.